initfiles/componentfiles/configxml/buildsetCC.xml.in

```diff
@@ -330,6 +330,12 @@
     </BuildSet>
     <BuildSet deployable="no"
              installSet="deploy_map.xml"
+             name="cassandraloggingagent"
```

Comment:
As we discussed, the existing ConfigMgr UI (can and) should be used to create a  Cassandra based Logging Agent. The part of configurations that the existing ConfigMgr UI does not support (for both  Cassandra based Logging Agent and MySQL based Logging Agent) is the LogGroup and the LogSourceMap. An Red Book item will be added to explain how to add them into environment.xml without using ConfigMgr.